tss-esapi/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "tss-esapi"
+version = "4.0.10-alpha.2"
+authors = ["Ionut Mihalcea <ionut.mihalcea@arm.com>",
+           "Hugues de Valon <hugues.devalon@arm.com>"]
+edition = "2018"
+description = "Rust-native wrapper around TSS 2.0 Enhanced System API"
+readme = "README.md"
+keywords = ["tpm", "tss", "esys", "esapi"]
+categories = ["api-bindings", "external-ffi-bindings", "cryptography"]
+license = "Apache-2.0"
+repository = "https://github.com/parallaxsecond/rust-tss-esapi"
+documentation = "https://docs.rs/crate/tss-esapi"
+
+[dependencies]
+bitfield = "0.13.2"
+serde = { version = "1.0.115", features = ["derive"] }
+mbox = "0.5.0"
+log = "0.4.11"
+enumflags2 = "0.6.4"
+num-derive = "0.3.2"
+num-traits = "0.2.12"
+hostname-validator = "1.0.0"
+regex = "1.3.9"
+zeroize = { version = "1.1.0", features = ["zeroize_derive"] }
+tss-esapi-sys = { path = "../tss-esapi-sys" }
+
+[dev-dependencies]
+env_logger = "0.7.1"
+
+[build-dependencies]
+bindgen = "0.56.0"
```

Comment:
`bindgen` is optional on `generate-bindings`, right?

Reply:
`bindgen` is no longer needed in the `tss-esapi` crate*